script/run-pre-commit.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+set -eu
+
+exec "$@"
```

Comment:
What's the point of this script?

`set -e` just exists on error, but here is one one command anyway
`set -u` errors on unset variables but none of them is used here

Reply:
The point is that without it pre-commit cannot find mypy
```
  entry: mypy
```
fails with "unable to find mypy" (not copied)
whereas 
```
entry: run-pre-commit.sh mypy
```
Works.
I spent a good hour trying to understand why. The other tools works perfect and mypy works perfect on the command line, just not in pre-commit.